paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/CompactAction.java

```diff
@@ -59,6 +59,8 @@ public class CompactAction extends TableActionBase {
 
     @Nullable private Duration partitionIdleTime = null;
 
+    private Boolean fullCompaction;
```

Comment:
Why not define a enum of CompactionType?

Reply:
Refer to above, If there are more other strategies in the future, we can use enumeration.